src/js/components/HistoryToolbar.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import {HistoryList} from './history/HistoryList';
+
+export class HistoryToolbar extends React.PureComponent<{}, {modalIsOpen: boolean}> {
```

Comment:
this component only wraps another component. merge into one `HistoryList` component?

Reply:
+1